accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerReindexer.scala

```diff
@@ -1,41 +1,91 @@
 package geotrellis.spark.io.accumulo
 
-import geotrellis.spark.{LayerId, Boundable}
-import geotrellis.spark.io.avro._
-import geotrellis.spark.io.index.KeyIndexMethod
+import geotrellis.spark._
 import geotrellis.spark.io._
+import geotrellis.spark.io.avro._
+import geotrellis.spark.io.index._
+import geotrellis.spark.io.json._
+
+import org.apache.avro.Schema
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
+import org.joda.time.DateTime
 import spray.json.JsonFormat
+
 import scala.reflect.ClassTag
 
 object AccumuloLayerReindexer {
-  def apply[K: Boundable: AvroRecordCodec: JsonFormat: ClassTag, V: AvroRecordCodec: ClassTag, M: JsonFormat](
+  def apply(
+    instance: AccumuloInstance,
+    attributeStore: AttributeStore[JsonFormat],
+    options: AccumuloLayerWriter.Options
+  )(implicit sc: SparkContext): AccumuloLayerReindexer =
+    new AccumuloLayerReindexer(instance, AccumuloAttributeStore(instance), options)
```

Comment:
Disregarding `attributeStore` argument here and creating a new one form the instance here.